Mathlib/Data/Set/Prod.lean

```diff
@@ -299,6 +301,9 @@ theorem fst_image_prod (s : Set β) {t : Set α} (ht : t.Nonempty) : Prod.fst ''
     let ⟨x, hx⟩ := ht
     ⟨(y, x), ⟨hy, hx⟩, rfl⟩
 
+lemma mapsTo_fst_prod {s : Set α} {t : Set β} : MapsTo Prod.fst (s ×ˢ t) s :=
```

Comment:
presumably `t` should be explicit here?
```suggestion
lemma mapsTo_fst_prod {s : Set α} (t : Set β) : MapsTo Prod.fst (s ×ˢ t) s :=
```

Reply:
I don't think so: in most cases, this should be used to clear side goals where everything is already known, and in the other cases one can use the `(t := foo)` syntax. That's quite different from Lean 3, where it should definitely have been explicit.